.changeset/two-apricots-serve.md

```diff
@@ -0,0 +1,7 @@
+---
+"@ledgerhq/live-common": minor
+"ledger-live-desktop": patch
```

Comment:
Can you update your changeset please ? You should use `minor` here, we only use `patch` when we do hotfixes
See this [thread](https://ledger.slack.com/archives/C0308JT8WS2/p1732007985801419)